server/semantics-service/src/main/java/edu/illinois/ncsa/incore/service/semantics/model/Type.java

```diff
@@ -19,22 +20,28 @@ public class Concept {
 
     private Columns tableSchema;
 
+    @Property("openvocab:versionnumber")
+    private String version;
+
+    @Property("dc:description")
+    private String description;
+
     // TODO how to represent @context
```

Comment:
We need to add context. We have `Context.java` in the model folder, but it needs to be updated with below. Not sure how we can represent this data structure with list of "string" and "dictionary"
```
 "@context": [
        "https://www.w3.org/ns/csvw#",
        {
            "@language": "en",
            "dc": "https://purl.org/dc/terms/",
            "gml": "https://schemas.opengis.net/gml/",
            "wfs": "https://schemas.opengis.net/wfs/1.1.0/wfs.xsd",
            "xlink": "https://www.w3.org/1999/xlink/",
            "xsd": "https://www.w3.org/2001/XMLSchema#",
            "qudt": "https://qudt.org/schema/qudt/",
            "unit": "https://qudt.org/vocab/unit/",
            "openvocab": "https://vocab.org/open/",
            "rdf": "https://www.w3.org/1999/02/22-rdf-syntax-ns#",
            "rdfs": "https://www.w3.org/2000/01/rdf-schema#",
            "dcat": "https://www.w3.org/ns/dcat#",
            "prov": "https://www.w3.org/ns/prov#"
        }
    ],
 ```

Reply:
For now, I simply use `List<?>` for @context so this way it's a list with different types of elements. I tried it with String and Map, but it is not doable if there is no key for each item. 

We may consider changing the format to the following to make it standard. However, if we only do simple read and write, I think what we have now is sufficient and don't need to change the format.
```
"@context": [
        "namespace" : "https://www.w3.org/ns/csvw#",
        "vocabularies" : {
            "@language": "en",
            "dc": "https://purl.org/dc/terms/",
            "gml": "https://schemas.opengis.net/gml/",
            "wfs": "https://schemas.opengis.net/wfs/1.1.0/wfs.xsd",
            "xlink": "https://www.w3.org/1999/xlink/",
            "xsd": "https://www.w3.org/2001/XMLSchema#",
            "qudt": "https://qudt.org/schema/qudt/",
            "unit": "https://qudt.org/vocab/unit/",
            "openvocab": "https://vocab.org/open/",
            "rdf": "https://www.w3.org/1999/02/22-rdf-syntax-ns#",
            "rdfs": "https://www.w3.org/2000/01/rdf-schema#",
            "dcat": "https://www.w3.org/ns/dcat#",
            "prov": "https://www.w3.org/ns/prov#"
        }
    ],
```

Reply:
woah nice it's getting closer. But the tricky thing is this @context might have meaning in semantic world. Is there possibility we can some how maneuver it to the same "key" (order differences might be ok). 

If java model doesn't work, maybe right before we send out "response" can we do some manipulation there?

@navarroc knows more about Java, any workrounds you can think of?

Reply:
I see. Or, we can represent this context as Document as before?

Reply:
I just added a function to construct output (modify "context" to "@context"). Here is the output so far:
```
{
    "license": {
        "@id": "http://opendefinition.org/licenses/cc-by/"
    },
    "description": "",
    "id": "60ba73c58c732f21246e7a44",
    "title": "ergo:annualGrossSalesOrProduction",
    "@context": [
        "http://www.w3.org/ns/csvw",
        {
            "@language": "en",
            "gml": "http://www.opengis.net/gml/",
            "iwfs": "http://www.ionicsoft.com/wfs/",
            "xlink": "http://www.w3.org/1999/xlink/",
            "xsd": "http://www.w3.org/2001/XMLSchema/",
            "qudt": "http://qudt.org/2.0/schema/qudt/",
            "unit": "http://qudt.org/vocab/unit/",
            "openvocab": "http://vocab.org/open/"
        }
    ],
    "version": "1.0",
    "url": "ergo:annualGrossSalesOrProduction",
    "tableSchema": {
        "columns": [
            {
                "name": "number",
                "titles": "Number",
                "description": "",
                "datatype": "xsd:integer",
                "required": "true",
                "unit": ""
            },
            {
                "name": "occupancy",
                "titles": "Occupancy",
                "description": "",
                "datatype": "xsd:string",
                "required": "true",
                "unit": ""
            },
            {
                "name": "occ_desc",
                "titles": "Occupancy Description",
                "description": "",
                "datatype": "xsd:string",
                "required": "true",
                "unit": ""
            },
            {
                "name": "output_emp",
                "titles": "1990 Output / Employment",
                "description": "",
                "datatype": "xsd:double",
                "required": "true",
                "unit": "unit:dollar"
            },
            {
                "name": "emp_sqft",
                "titles": "Sq. ft floor Space / Employee",
                "description": "",
                "datatype": "xsd:double",
                "required": "true",
                "unit": ""
            },
            {
                "name": "annual_sal",
                "titles": "Annual Sales ($/sq.ft.)",
                "description": "",
                "datatype": "xsd:double",
                "required": "true",
                "unit": ""
            }
        ]
    }
}
```